heartcheck.gemspec

```diff
@@ -27,4 +27,5 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'rack-test', '~> 0.6.0', '>= 0.6.3'
   # for documentation
   spec.add_development_dependency 'yard', '~> 0.8.0', '>= 0.8.7.6'
+  spec.add_development_dependency 'concurrent-ruby', '~> 0.9.1', '>= 0.9.1'
```

Comment:
With this approach we will force installation of 'concurrent' gems just for users that really want use this feature, and users who don't care dont need to get the 'extra' gems. WDYT?